tests/module_api/test_api.py

```diff
@@ -532,6 +534,38 @@ def test_update_membership(self):
         self.assertEqual(res["displayname"], "simone")
         self.assertIsNone(res["avatar_url"])
 
+        # Check that no remote join attempts have occurred thus far.
```

Comment:
Given the amount of extra code this should probably be its own test (eg `test_update_membership_remote_join`)

Reply:
Done: 30bc9e5